parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -295,7 +295,7 @@ retrycmd_oras_login() {
     for i in $(seq 1 $retries); do
         ORAS_LOGIN_OUTPUT=$(oras login "$acr_url" --identity-token-stdin --registry-config "${ORAS_REGISTRY_CONFIG_FILE}" <<< "$REFRESH_TOKEN" 2>&1)
         exit_code=$?
-        if [[ $exit_code -eq 0 ]]; then
+        if [ "$exit_code" -eq 0 ]; then
```

Comment:
oh, we can use -eq with strings and ints interchangeably?

Reply:
yes. -eq is smart to handle that. 

```
root@DESKTOP-1TJD8FB:~/git/AgentBaker# exit_code=0

if [ "$exit_code" -eq 0 ]; then
    echo "The command succeeded."
else
    echo "The command failed."
fi
```
Output: `The command succeeded.`